styles/variables.css

```diff
@@ -16,4 +16,6 @@
     --info-border:#bee5eb;
     --info-text:#0c5460;
     --black-default:black;
+    --green-default:green;
```

Comment:
please use hex codes for black and green colors

Reply:
i missed it.. I will update that